src/development/tools/ide/android-studio.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: Developing Flutter apps using Android Studio / IntelliJ
+long-title: Developing Flutter apps using Android Studio / IntelliJ # Not currently used
+title: Android Studio / IntelliJ
```

Comment:
I've shortened the title. Is this ok @Sfshaza? (If so then I'll remove the `long-title` which I was using to hold the old longer title temporarily.)

Reply:
Sure